aii-opennebula/src/main/resources/network_ar.tt

```diff
@@ -11,7 +11,9 @@ IP = "[% system.network.interfaces.${tmpbridge}.ip %]",
 [%         ELSE -%]
 IP = "[% system.network.interfaces.${pair.key}.ip %]",
 [%         END -%]
+[%         IF hardware.cards.nic.${pair.key}.hwaddr.defined -%]
```

Comment:
but this is not optional in the hardware schema, so will always be present. we  need to set a private MAC and somehow define that if this MAC is set, it is to be ignored in the ONE template.